lib/email_user_extensions.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module EmailUserExtensions
```

Comment:
This seems like a name some other plugins might use... I wonder if we should namespace all the `_extensions` classes like

```suggestion
module SavedSearches::EmailUserExtensions
```
(and similarly for `GuardianExtensions`, `UserExtensions` and `UserNotificationExtensions`.